Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -303,5 +319,21 @@ public override IAccountStateDelta Execute(IActionContext context)
                     .SetState(BlockIndexKey, context.BlockIndex);
             }
         }
+
+        private class InvalidAction : BaseAction
```

Comment:
It would be nice if the class name is more self-descriptive.  There are many ways for `IAction` implementation to be invalid.  I suggest the name `ActionNotAttributeAnnotated`.